events/station-login.js

```diff
@@ -0,0 +1,55 @@
+function stationLogin(event, payload){
+
+    // Unload the payload into a constant
+    const {station} = payload;
+
+    // Get the gamestate
+    const game = context.getGameState();
+
+    // Get the station- and playerobject so that we can get their attributes
+    const stationObj = game.stations[station];
+    const player = game.players[context.id()];
+
+    //Some error checking
+    // We're not in a game
+    if (game === null) {
+        context.send('event:station:login:fail', { errorCode: 0 });
+        return;
+      }
+    // We're not in the play phase
+    if (game.properties.phase.type !== 2) {
+    context.send('event:station:login:fail', { errorCode: 1 });
```

Comment:
Did you run `npx eslint . --quiet`? Seems like the indentation is a bit off here.